.github/workflows/test.yml

```diff
@@ -56,6 +56,17 @@ jobs:
         --config=clang
         -c opt
         $(bazel query 'kind(was.*_rust_binary, //test/test_data/...)')
+        $(bazel query 'kind(_optimized_wasm_cc_binary, //test/test_data/...)')
+
+    - name: Clean up test data
```

Comment:
Why do we need to do it? Are the runfiles corrupted due to changing between compilation modes? If so, it might be worth adding the comment explaining why.

Reply:
When using `proxy_wasm_cc_binary`, the final artifact seems to have `_optimized_wasm_cc_binary` kind.

```
$ bazel query //test/test_data/... --output=label_kind 2>&1 | grep configure_check
_optimized_wasm_cc_binary rule //test/test_data:configure_check.wasm
cc_binary rule //test/test_data:proxy_wasm_configure_check
wasm_cc_binary rule //test/test_data:wasm_configure_check.wasm
```

Will add the comment here.

Reply:
Sorry, I didn't mean the `bazel query`, I asked about why do we need to clean up test data.

The comment about `_optimized_wasm_cc_binary` is pretty useless, feel free to remove it (and if not, then it should be in the correct step, not in the clean up).

Reply:
Done.